lms/templates/admin/installation.html.jinja2

```diff
@@ -0,0 +1,93 @@
+{% extends "lms:templates/admin/base.html.jinja2" %}
+{% block header %}
+Installation {{ installation.id }} / {{installation.consumer_key}}
```

Comment:
I'm calling this installation here, Application Instance it's not a great name but just "Consumer key" might be a bit reductive.

Happy to rename back to something else thought.

Reply:
Hmm. Might be worth some wider discussion in Slack?

* "Application instance" is clearly terrible, although it is what's already used in the DB and source and known by support staff
* "Installation" is better but I think it's actually incorrect, since a single `ApplicationInstance` can be installed multiple times in multiple LMS's, or I think even multiple times in the _same_ LMS, or not installed anywhere at all. For example changing the settings of an `ApplicationInstance` will change the settings for _all_ installations that use that consumer key
* Personally I think "consumer key" is correct as that's what it actually is: a consumer key and related fields (shared secret and other stuff)
* Maybe there's another name that better captures that this isn't _just_ the consumer key but also other related fields and settings associated with this consumer key?

Reply:
Renamed to ApplicationInstance